src/transformers/processing_utils.py

```diff
@@ -993,6 +993,58 @@ def apply_chat_template(
         )
 
 
+def _validate_images_text_input_order(images, text):
+    """
+    For backward compatibility: reverse the order of `images` and `text` inputs if they are swapped.
+    This method should only be called for processors where `images` and `text` have been swapped for uniformization purposes.
+    Note that this method assumes that two `None` inputs are valid inputs. If this is not the case, it should be handled
+    in the processor's `__call__` method before calling this method.
+    """
+
+    def _is_valid_text_input_for_processor(t):
+        if isinstance(t, str):
+            # Strings are fine
+            return True
+        elif isinstance(t, (list, tuple)):
+            # List are fine as long as they are...
+            if len(t) == 0:
+                # ... not empty
+                return False
+            elif isinstance(t[0], (str, int)):
+                # ... list of strings or int (for encoded inputs)
+                return True
+            elif isinstance(t[0], (list, tuple)):
+                # ... list of list of strings or int (for list of encoded inputs)
+                if isinstance(t[0][0], (str, int)):
+                    return True
+                elif isinstance(t[0][0], (list, tuple)):
+                    # ... list of list of list of int (for list of list of encoded inputs)
+                    return isinstance(t[0][0][0], int)
```

Comment:
Reminder to remove the int checks in this function

Reply:
Oops, should be good now